src/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  position: relative;
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.info {
+  padding: auto;
+}
```

Comment:
What is info? And why it's separate BEM block? check it please, maybe you don't need it